python/src/usecases/stiffened_panel.py

```diff
@@ -0,0 +1,130 @@
+"""
+Use case : stiffened panel
+==========================
+"""
+import openturns as ot
+
+
+class StiffenedPanel:
+    r"""
+    Data class for the stiffened panel model.
+
+
+    Attributes
+    ----------
+
+    dim : Constant, the dimension of the problem.
+        dim=10
+
+    model : `SymbolicFunction`
+        Model of the critical shearing load.
+        The model has input dimension 10 and output dimension 1.
+        More precisely, we have :math:`\vect{X} = (E, \nu, h_c, \ell, f_1, f_2, t, a, b_0, p)`
+        and  :math:`Y = ((N_{xy})_{cr})`.
```

Comment:
This is a real number, which does not require outer parentheses.
```suggestion
        and  :math:`Y = (N_{xy})_{cr}`.
```